ktor-server/ktor-server-core/mingwX64/test/BuildApplicationConfigNixTest.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2014-2021 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+import io.ktor.server.engine.*
+import platform.posix.*
+import kotlin.test.*
+
+class BuildApplicationConfigNixTest {
+
+//    @Test
+//    fun testPropertyConfig() {
```

Comment:
The only thing that is missing it this test. There is no `getenv`/`setenv` for mingw out-of-the-box.
And without Windows PC I can't test it properly

Reply:
So what about this? https://github.com/ktorio/ktor/blob/20cb0455457e86ea5cf01fb90ccaa945932317b2/ktor-server/ktor-server-config-yaml/posix/src/io/ktor/server/config/yaml/YamlConfigNix.kt#L75

Reply:
Can you use `putenv`?

Reply:
I found online that `putenv` doesn't copy pointer which is passed but just stores it.
On current moment it accepts `String`, this means K/N cinterop creates temporary pointer for the duration of a bridge call.
So I can try to use it, but can't test it, so may be someone else could do it :)

Reply:
> So what about this?

Oh, sorry, there is no `setenv`/`unsetenv`...

Reply:
That sounds pretty bad, but I am not too familiar with this. Maybe https://learn.microsoft.com/en-us/windows/win32/api/processenv/nf-processenv-setenvironmentvariablew can be used.

Reply:
Thanks!
used `SetEnvironmentVariable` - let's see what CI will say (I absolutely love CI-driven development)